### PR TITLE
AOW-32 Using aws s3 to delete file 

### PR DESCRIPTION
**Pull request by FrankQixiangGao**

### CODE REVIEW

File: controller/files.js [hunk 1]
```diff
@@ -1,7 +1,18 @@
 const GfsFiles = require('../service/gfsFiles');
+const NotFoundError = require('../model/error/not-found');
 
 exports.deleteOneFile = async (req, res, next, dbClient) => {
```
Comment (FrankQixiangGao) [inline controller/files.js]: delete async

File: service/gfsFiles.js [hunk 2]
```diff
@@ -1,10 +1,17 @@
-const gridfsBucket = require('../client/gridfsBucket');
 // const Storage = require('./storage');
 const client = require('../client/mongodb');
-const NotFoundError = require('../model/error/not-found');
 const BadRequestError = require('../model/error/bad-request');
+const aws = require('aws-sdk');
```
Comment (FrankQixiangGao) [inline service/gfsFiles.js]: move aws to top of the file

File: service/gfsFiles.js [hunk 3]
```diff
@@ -1,10 +1,17 @@
-const gridfsBucket = require('../client/gridfsBucket');
 // const Storage = require('./storage');
```
Comment (FrankQixiangGao) [inline service/gfsFiles.js]: delete it

**Comment by Jackiebibili** [comment]
lgtm

**Review by Jackiebibili — APPROVED** [review]
lgtm

**Pull request merged**